src/command_modules/azure-cli-backup/setup.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup
+
+VERSION = "0.0.1b1+dev"
+
+# The full list of classifiers is available at
+# https://pypi.python.org/pypi?%3Aaction=list_classifiers
+CLASSIFIERS = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Developers',
+    'Intended Audience :: System Administrators',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 2.7',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+]
+
+DEPENDENCIES = [
+    'azure-mgmt-recoveryservices==0.1.0',
+    'azure-mgmt-recoveryservicesbackup==0.1.0',
+    'azure-cli-core',
+    'azure-cli-vm'
```

Comment:
Command modules should not depend on each other `azure-cli-vm`.
Is this used anywhere?